============================= test session starts ==============================
platform linux -- Python 3.8.12, pytest-6.2.5, py-1.11.0, pluggy-1.0.0
rootdir: /var/lib/jenkins/vision, configfile: pytest.ini
plugins: hypothesis-4.53.2
collected 1212 items

test/test_transforms.py ................................................ [  3%]
..............ssss...................................................... [  9%]
........................................................................ [ 15%]
........................................................................ [ 21%]
........................................................................ [ 27%]
........................................................................ [ 33%]
........................................................................ [ 39%]
..........s............................................................. [ 45%]
........................................................................ [ 51%]
........................................................................ [ 57%]
........................................................................ [ 63%]
........................................................................ [ 69%]
........................................................................ [ 75%]
........................................................................ [ 81%]
........................................................................ [ 87%]
........................................................................ [ 93%]
........................................................................ [ 99%]
............                                                             [100%]

=============================== warnings summary ===============================
<frozen importlib._bootstrap>:219
  <frozen importlib._bootstrap>:219: ImportWarning: can't resolve package from __spec__ or __package__, falling back on __name__ and __path__

test/test_transforms.py::test_adjust_gamma
test/test_transforms.py::test_adjusts_L_mode
  /opt/conda/lib/python3.8/site-packages/PIL/Image.py:1680: DeprecationWarning: an integer is required (got type float).  Implicit conversion to integers using __int__ is deprecated, and may be removed in a future version of Python.
    return self._new(self.im.point(lut, mode))

-- Docs: https://docs.pytest.org/en/stable/warnings.html
=========================== short test summary info ============================
SKIPPED [1] test/test_transforms.py:156: accimage not available
SKIPPED [1] test/test_transforms.py:164: accimage not available
SKIPPED [1] test/test_transforms.py:173: accimage not available
SKIPPED [1] test/test_transforms.py:193: accimage not available
SKIPPED [1] test/test_transforms.py:924: Temporarily disabled
================= 1207 passed, 5 skipped, 3 warnings in 28.11s =================
